redhat-cop/helm-charts
# Support ImagePullSecret for pulling from private registry

#### What is this PR About?
Adds the ability to specify an image pull secret for pulling a custom image from a private container registry

#### How do we test this?
It should run as-is if pulling the default image, since it only adds the secret if `installPlanApproverAndVerifyJobsImagePullSecret` is specified.  

Test that the secret works requires

- Update values.yaml `installPlanApproverAndVerifyJobsImage` to point at to an image in a private registry
- Create an imagepull secret with credentials for the private registry
- Update values.yaml `installPlanApproverAndVerifyJobsImagePullSecret`

cc: @redhat-cop/day-in-the-life
